clsCommon.pri

```diff
@@ -32,6 +39,16 @@ HEADERS *=\
 	source/dataman/info/CLSOceanOptics65000DetectorInfo.h \
 	source/dataman/info/CLSAmptekSDD123DetectorInfo.h \
 	source/dataman/info/CLSSynchronizedDwellTimeConfigurationInfo.h \
+	source/ui/CLS/CLSValueProgressLabel.h \
+	source/ui/CLS/CLSValueEditor.h \
+	source/ui/CLS/CLSControlEditor.h \
```

Comment:
Is it time to think about doing the full replace of AMControlEditor with your better version (as a separate issue) @helfrij?